src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -19,7 +19,7 @@
             + "Contact details can be marked private by prepending 'p' to the prefix.\n"
             + "Parameters: NAME [p]p/PHONE [p]e/EMAIL [p]a/ADDRESS  [t/TAG]...\n"
             + "Example: " + COMMAND_WORD
-            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
+            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25, 231534 t/friends t/owesMoney";
```

Comment:
It's nice that you also update the help message :)
How about the documentation (e.g. user guide) ?